src/codemagic/tools/firebase/action_groups/firebase_action_groups.py

```diff
@@ -0,0 +1,8 @@
+from codemagic import cli
+
+
+class FirebaseActionGroups(cli.ActionGroup):
+    RELEASES = cli.ActionGroupProperties(
+        name='releases',
+        description='Manage your Firebase releases',
```

Comment:
IDK if there are any other releases in Firebase. Maybe we should be a tiny bit more to the point here?
```suggestion
        description='Manage your Firebase app releases',
```